packages/web/curl/package.mk

```diff
@@ -65,23 +65,17 @@ PKG_CONFIGURE_OPTS_TARGET="ac_cv_lib_rtmp_RTMP_Init=yes \
                            --with-random=/dev/urandom \
                            --with-gnutls \
                            --without-ssl \
-                           --without-polarssl \
+                           --without-mbedtls \
                            --without-nss \
                            --with-ca-bundle=/run/libreelec/cacert.pem \
                            --without-ca-path \
                            --without-libpsl \
                            --without-libmetalink \
                            --without-libssh2 \
-                           --with-librtmp=$SYSROOT_PREFIX/usr \
-                           --without-libidn \
-                           --without-libidn2 \
+                           --with-librtmp \
+                           --with-libidn2 \
```

Comment:
should probably depend on libidn2

Reply:
Thanks, had it right in my build branch.